cfgov/v1/forms.py

```diff
@@ -175,71 +236,42 @@ def set_field_html_name(self, field, new_name):
             field.widget.render = lambda name, value, attrs=None: \
                 old_render(new_name, value, attrs)
 
-    # Generates a query by iterating over the zipped collection of
-    # tuples.
-    def generate_query(self):
-        final_query = Q()
-        if self.is_bound:
-            for query, field_name in zip(self.get_query_strings(), self.declared_fields):
-                if self.cleaned_data.get(field_name):
-                    final_query &= \
-                        Q((query, self.cleaned_data.get(field_name)))
-        return final_query
-
-    # Returns a list of query strings to associate for each field, ordered by
-    # the field declaration for the form. Note: THEY MUST BE ORDERED IN THE
-    # SAME WAY AS THEY ARE DECLARED IN THE FORM DEFINITION.
-    def get_query_strings(self):
-        return [
-            'title__icontains',      # title
-            'date_published__gte',   # from_date
-            'date_published__lte',   # to_date
-            'categories__name__in',  # categories
-            'tags__slug__in',        # topics
-            'authors__slug__in',     # authors
-        ]
-
 
 class EventArchiveFilterForm(FilterableListForm):
-    def get_query_strings(self):
-        return [
-            'title__icontains',      # title
-            'start_dt__gte',         # from_date
-            'end_dt__lte',           # to_date
-            'categories__name__in',  # categories
-            'tags__slug__in',        # topics
-            'authors__slug__in',     # authors
-        ]
-
+    def __init__(self, *args, **kwargs):
+        super(EventArchiveFilterForm, self).__init__(*args, **kwargs)
+        self.query_type = 'event-archive'
```

Comment:
i know theres only 2 "query types", but if this list does grow, or just to make it easier for future people to now what the available options are for this, you could move it to an enum class - that lists what the possible values might be.